services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -326,10 +328,12 @@ export class NOWPermitGeneration extends Component {
         const fieldIdParts = key.split(/_(.+)/);
         const guid = fieldIdParts[0];
         const fieldName = fieldIdParts[1];
-        if (!(guid in allFileMetadata)) {
-          allFileMetadata[guid] = {};
+        if (guid && guid !== "null") {
```

Comment:
So this doesn't add the doc in the request if the guid is null, but isn't it a problem that the guid is null in the first place?

Reply:
It is, and this is a band-aid fix for sure. I didn't find the root cause as to why it's happening and got stuck at "redux forms magic".

Reply:
What if this causes a different more worse problem? :( Idk I haven't wrapped my head around it yet. Might be good to get someone else to look at it and get their opinion and maybe have a team discussion? @mhuseinov @mmumby

Reply:
*Resolved in convo with team*